test/t/test_help.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+
+class TestHelp:
+    @pytest.mark.complete("help ")
+    def test_basic(self, completion):
+        assert "help" in completion
+
+    @pytest.mark.complete("help -")
+    def test_options(self, completion):
+        assert completion
+
+    @pytest.mark.complete(r"help \(")
+    def test_parens(self, completion):
+        # Assumption: an item like "(( ... ))" exists in the output
+        assert any(
+            x.startswith(r"\(") and x.endswith(r"\)\)") for x in completion
```

Comment:
bash-4.2 doesn't seem to produce `(( ... ))` for `compgen -A helptopic`:

```bash
$ bash-4.3 -c 'compgen -A helptopic' | grep '('
(( ... ))
for ((
$ bash-4.2 -c 'compgen -A helptopic' | grep '('
$
```

Reply:
7b21a8f167e6ab28c2c4d399909cf9ce5155eec9